docs/synapse/deploymentguide.rst

```diff
@@ -0,0 +1,355 @@
+
+Deploy JSONStor Service
+=======================
+
+**Inside the AHA container**
+
+Generate a one-time use provisioning URL::
+
+    python -m synapse.tools.aha.provision.service 00.jsonstor
+
+You should see output that looks similar to this::
+
+    one-time use URL: ssl://aha.loop.vertex.link:27272/<guid>?certhash=<sha256>
+
+**On the Host**
+
+Create the container directory::
+
+    mkdir -p /srv/syn/00.jsonstor/storage
+    chown -R 999 /srv/syn/00.jsonstor/storage
+
+Create the ``/srv/syn/00.jsonstor/docker-compose.yaml`` file with contents::
+
+    version: "3.3"
+    services:
+      00.jsonstor:
+        user: 999
+        image: vertexproject/synapse-jsonstor:v2.x.x
+        network_mode: host
+        restart: unless-stopped
+        volumes:
+        - ./storage:/vertex/storage
+        environment:
+            - SYN_JSONSTOR_AHA_PROVISION=ssl://aha.loop.vertex.link:27272/<guid>?certhash=<sha256>
+
+.. note::
+
+    Don't forget to replace your one-time use provisioning URL!
+
+Start the container::
+
+    docker-compose --file /srv/syn/00.jsonstor/docker-compose.yaml pull
+    docker-compose --file /srv/syn/00.jsonstor/docker-compose.yaml up -d
+
+Deploy Cortex Service
+=====================
+
+**Inside the AHA container**
+
+Edit or copy the following contents to the file ``/tmp/cortex.yaml`` inside the container::
+
+    axon: aha://axon...
+    jsonstor: aha://jsonstor...
+
+Generate a one-time use provisioning URL::
+
+    python -m synapse.tools.aha.provision.service 00.cortex --user root --cellyaml /tmp/cortex.yaml
+
+You should see output that looks similar to this::
+
+    one-time use URL: ssl://aha.loop.vertex.link:27272/<guid>?certhash=<sha256>
+
+**On the Host**
+
+Create the container directory::
+
+    mkdir -p /srv/syn/00.cortex/storage
+    chown -R 999 /srv/syn/00.cortex/storage
+
+Create the ``/srv/syn/00.cortex/docker-compose.yaml`` file with contents::
+
+    version: "3.3"
+    services:
+      00.cortex:
+        user: 999
+        image: vertexproject/synapse-cortex:v2.x.x
+        network_mode: host
+        restart: unless-stopped
+        volumes:
+        - ./storage:/vertex/storage
+        environment:
+            - SYN_CORTEX_AHA_PROVISION=ssl://aha.loop.vertex.link:27272/<guid>?certhash=<sha256>
+
+.. note::
+
+    Don't forget to replace your one-time use provisioning URL!
+
+Start the container::
+
+    docker-compose --file /srv/syn/00.cortex/docker-compose.yaml pull
+    docker-compose --file /srv/syn/00.cortex/docker-compose.yaml up -d
+
+Remember, you can view the container logs in real-time using::
+
+    docker-compose --file /srv/syn/00.cortex/docker-compose.yaml logs -f
+
+Deploy Cortex Mirror (optional)
+===============================
+
+**Inside the AHA container**
+
+Generate a one-time use URL for provisioning from *inside the AHA container*::
+
+    python -m synapse.tools.aha.provision.service 01.cortex --mirror 00.cortex
```

Comment:
Similar to the upstream I had to use ``--user root`` and then also replace the cortex user in the cell.yaml after initial boot.  Also, I had to specify the network name, e.g. 00.cortex.loop.vertex.link